ofl/lohitbengali/METADATA.pb

```diff
@@ -13,3 +13,4 @@ fonts {
   copyright: "Copyright 2011 Lohit Fonts Project contributors (psatpute@redhat.com). <http://fedorahosted.org/lohit> Licensed under the SIL Open Font License 1.1 (see file OFL.txt)"
 }
 subsets: "bengali"
+primary_script: "Beng"
```

Comment:
The three LOHIT families are not in the live catalog, maybe the script should only use the family list from the live server.

Reply:
Is it a problem to add this information to non-live fonts?

Reply:
We generate the lists using all modified directories path so yes cause they might be pushed by mistake.

Reply:
I forgotten why lohit not added, is it no Latin?